app/views/teachers/mailer/confirmation_instructions.html.erb

```diff
@@ -2,4 +2,4 @@
 
 <p>You can confirm your account email through the link below:</p>
 
-<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
+<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token, host: 'localhost', port: 3000) %></p>
```

Comment:
kindly put the `host` and `port` in some env variable instead of putting them directly in views. In that way, we can have dynamic values for `host` and `port`